content/en/examples/pods/storage/pv-pod.yaml

```diff
@@ -3,18 +3,16 @@ kind: Pod
 metadata:
   name: task-pv-pod
 spec:
-  volumes:
-    - name: task-pv-storage
-      persistentVolumeClaim:
-        claimName: task-pv-claim
   containers:
-    - name: task-pv-container
-      image: nginx
-      ports:
-        - containerPort: 80
-          name: "http-server"
-      volumeMounts:
-        - mountPath: "/usr/share/nginx/html"
-          name: task-pv-storage
-
-
+  - image: nginx
```

Comment:
It is better to place volumes below the container spec, but also in container spec name key should be first before image key.